src/Nethermind/Nethermind.Core/Deposit.cs

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Text;
+using System.Text.Json.Serialization;
+using Nethermind.Core.Extensions;
+using Nethermind.Int256;
+
+namespace Nethermind.Core;
+
+/// <summary>
+/// Represents a Deposit that has been validated at the consensus layer.
+/// </summary>
+public class Deposit
+{
+    public byte[]? PublicKey { get; set; }
```

Comment:
Is there Engine API spec for this EIP? Names of fields matter for json serialization, here: https://eips.ethereum.org/EIPS/eip-6110 PubKey was used, but the correct names should come from Engine API